bundle/Controller/Admin/DashboardController.php

```diff
@@ -39,6 +40,46 @@ public function indexAction(EntityManagerInterface $entityManager): array
         $repoBroadcast = $entityManager->getRepository(Broadcast::class);
         $repoUsers     = $entityManager->getRepository(User::class);
 
+        $translator = $this->container->get('translator');
+        $aUsers = $repoUsers->findLastUpdated();
+        foreach($aUsers as $user){
+            switch($user->getStatus()){
+
+                case $user::PENDING :
+                    $translatedTag =  $translator->trans('dashboard.users.status.pending', array(), 'ezmailing');
+                    $user->statusStyle = $user::STATUSES_STYLE[$user::PENDING];
```

Comment:
all the block is not useful. see comment ahead.

Also  `$user::STATUSES_STYLE` is invalid, please do `User::STATUSES_STYLE`